packages/ripple-keypairs/src/signing-schemes/ed25519/index.ts

```diff
@@ -0,0 +1,57 @@
+import { ed25519 as nobleEd25519 } from '@noble/curves/ed25519'
+import { bytesToHex } from '@xrplf/isomorphic/utils'
+
+import { HexString, SigningScheme } from '../../types'
+import assert from '../../utils/assert'
+import Sha512 from '../../utils/Sha512'
+
+const ED_PREFIX = 'ED'
+
+const ed25519: SigningScheme = {
+  deriveKeypair(entropy: Uint8Array): {
+    privateKey: string
+    publicKey: string
+  } {
+    const rawPrivateKey = Sha512.half(entropy)
+    const privateKey = ED_PREFIX + bytesToHex(rawPrivateKey)
+    const publicKey =
+      ED_PREFIX + bytesToHex(nobleEd25519.getPublicKey(rawPrivateKey))
+    return { privateKey, publicKey }
+  },
+
+  sign(message: Uint8Array, privateKey: HexString): string {
+    // noinspection SuspiciousTypeOfGuard
```

Comment:
What does this comment mean?

Reply:
Ah, it's intellij specific. It's telling it that `messageof instanceof Uint8Array` is actually needed, and that you can't trust actually trust the declared type at runtime. 

It can be removed. Will do.